RNTester/RNTesterPods.xcodeproj/xcshareddata/xcschemes/RNTester-macOS Release.xcscheme

```diff
@@ -99,14 +99,23 @@
       allowLocationSimulation = "YES">
       <BuildableProductRunnable
          runnableDebuggingMode = "0">
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "9F15345A233AB2C4006DFE44"
+            BuildableName = "RNTester-macOS.app"
+            BlueprintName = "RNTester-macOS"
+            ReferencedContainer = "container:RNTesterPods.xcodeproj">
+         </BuildableReference>
+      </BuildableProductRunnable>
+      <MacroExpansion>
```

Comment:
Just out of curiosity, do you know what this is about?

Reply:
It was auto added when I edited the release scheme...